<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OsagoTsBlank</class>
 <widget class="QDialog" name="OsagoTsBlank">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Сведения о ТС</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Марка, модель ТС</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1" colspan="2">
                <widget class="QLineEdit" name="modelLineEdit">
                 <property name="minimumSize">
                  <size>
                   <width>191</width>
                   <height>0</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Идентификационный номер</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1" colspan="2">
                <widget class="QLineEdit" name="vinLineEdit"/>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Мощность</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QSpinBox" name="powerSpinBox">
                 <property name="maximum">
                  <number>2000</number>
                 </property>
                </widget>
               </item>
               <item row="2" column="2">
                <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
                 <item>
                  <widget class="QRadioButton" name="kWtRadioButton">
                   <property name="text">
                    <string>КВт</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="hpRadioButton">
                   <property name="text">
                    <string>л.с.</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="countLabel">
                 <property name="text">
                  <string>Количество пассажирских 
мест</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QSpinBox" name="passengersCountSpinBox">
                 <property name="maximum">
                  <number>500</number>
                 </property>
                 <property name="value">
                  <number>8</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Категория ТС</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QComboBox" name="tsTypeComboBox">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <item>
                  <property name="text">
                   <string>A</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>B</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>C</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>D</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>E</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Год изготовления</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QSpinBox" name="yearSpinBox">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimum">
                  <number>1900</number>
                 </property>
                 <property name="maximum">
                  <number>2013</number>
                 </property>
                </widget>
               </item>
               <item row="2" column="0" colspan="2">
                <widget class="QLabel" name="massLabel">
                 <property name="text">
                  <string>Разрешенная максимальная масса (кг)</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="2">
                <widget class="QSpinBox" name="mass">
                 <property name="maximum">
                  <number>10000</number>
                 </property>
                 <property name="value">
                  <number>3500</number>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Шасси (рама) №</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Кузов (прицеп) №</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1" colspan="2">
                <widget class="QLineEdit" name="trackLineEdit"/>
               </item>
               <item row="3" column="1" colspan="2">
                <widget class="QLineEdit" name="shassiLineEdit">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Документ о регистрации ТС</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>Серия</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="serialLineEdit"/>
                </item>
                <item>
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>Номер</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="numLineEdit"/>
                </item>
                <item>
                 <widget class="QLabel" name="label_12">
                  <property name="text">
                   <string>Дата выдачи</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDateEdit" name="documentDateEdit"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QCheckBox" name="isRentalCheckBox">
             <property name="text">
              <string>ТС сдается в аренду</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Цель использования ТС</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="usingTypeComboBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <item>
              <property name="text">
               <string>Личная</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Учебная езда</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Такси</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Дорожные и специальные ТС</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Прочее</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="1">
          <widget class="QSpinBox" name="spinBox_5"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Класс, зависящий от наличия страховых выплат,
 присвоенный собственнику</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="classesComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Количество страховых случаев по предыдущему договору 
обязательного страхования владельцев данного ТС.</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="prevButton">
         <property name="text">
          <string>Назад</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>458</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="nextButton">
         <property name="text">
          <string>Далее</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
